<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>188</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Age :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_age_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>188</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Name :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Weight(Kg) :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_weight_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>122</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Height(cm):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_height_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Blood Pressure :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_bloodpressure"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>124</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_gender_2">
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UserData</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>personalData:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>health Data :</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>188</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>mail :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_mail_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>195</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>number of meals per day : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_height_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>148</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>diseases :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_diseases_2">
       <item>
        <property name="text">
         <string>Heart issues</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Low Pressure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>High Pressure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diabetes</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
